### PR TITLE
Create and populate define section

### PR DESCRIPTION
**Pull request by StackScribe**
This creates a new doc section to discuss the different functionality one can define in a Keptn project.  In this first stage, we move all the existing pages into this section with minimal modifications, although most of them need some work.

Also changed weights of several sections to get them organized appropriately.

The quality_gates section previously had multiple files in it.  Some were turned into reference pages and the two remaining files were combined into a single file in the define section.

### CODE REVIEW

File: content/docs/0.18.x/define/quality-gates/index.md [hunk 1]
```diff
@@ -1,24 +1,131 @@
 ---
-title: Working and Integrating with Quality Gates
-description: Integrate a Quality Gate into an existing pipeline.
-weight: 7
+title: Quality Gates
+description: Implement Quality Gates evaluations in your project
```
Comment (thschue) [inline content/docs/0.18.x/define/quality-gates/index.md:3]:
```suggestion
description: Implement Quality Gate evaluations in your project
```

File: content/docs/0.18.x/define/quality-gates/index.md [hunk 2]
```diff
@@ -1,24 +1,131 @@
...
+weight: 70
 keywords: [0.18.x-quality_gates]
 ---
 
+A quality gate allows you to validate a deployment or release
+using data delivered by your observability platform
```
Comment (thschue) [inline content/docs/0.18.x/define/quality-gates/index.md:9]:
```suggestion
using data provided by your observability platform
```

File: content/docs/0.18.x/define/quality-gates/index.md [hunk 3]
```diff
@@ -1,24 +1,131 @@
...
+and using quality criteria that you define as Service Level Objectives (SLOs).
```
Comment (thschue) [inline content/docs/0.18.x/define/quality-gates/index.md:10]: Not sure, but are SLOs defined somewhere?

File: content/docs/0.18.x/define/quality-gates/index.md [hunk 4]
```diff
@@ -1,24 +1,131 @@
...
+
+* To create the Keptn project `easyBooking`,
+use the [keptn create project](../../reference/cli/commands/keptn_create_project/) command
+as discussed in [Create a project](../../manage/project).
+
+* To create the Keptn service `booking`,
+use the [keptn create service](../../reference/cli/commands/keptn_create_service/) command
+as discussed in [Create a service](../service/#create-a-service).
+
+**Note:** It is not necessary to create the stage since it is declared in the Shipyard that is applied during the project creation.
```
Comment (thschue) [inline content/docs/0.18.x/define/quality-gates/index.md:69]: Should we also mention somewhere that stages can not be added or removed after the initial creation?

File: content/docs/0.18.x/define/quality-gates/index.md [hunk 5]
```diff
@@ -1,24 +1,131 @@
...
+
+* [Prometheus](https://artifacthub.io/packages/keptn/keptn-integrations/prometheus-service)
+
+**Step 3: Add SLO configuration to a service**
+
+By adding an [Service-Level Objective (SLO)](../../reference/files/slo/#service-level-objective) config to your service, you *activate* a quality gate for that service.
+
+* To add an SLO to the `booking` service, use the [keptn add-resource](../../reference/cli/commands/keptn_add-resource/) CLI command.
+
+**Step 4: Trigger the quality gate**
```
Comment (thschue) [inline content/docs/0.18.x/define/quality-gates/index.md:100]:
```suggestion
**Step 4: Use the quality gate**
```

**Review by thschue — CHANGES_REQUESTED** [review]

File: content/docs/0.18.x/define/quality-gates/index.md (hunk 4 above)
Comment (StackScribe) [reply content/docs/0.18.x/define/quality-gates/index.md:69]: Yes, that is mentioned in several places.  Actually, I think all of this basic create-service text that is in the Quality Gates page needs to be deleted and replaced with a link to the "Create a project" page under "Managing Projects."   After we get the files restructured, we can look and see where we need to reiterate that stages cannot be added or removed.

File: content/docs/0.18.x/define/quality-gates/index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/0.18.x/define/quality-gates/index.md:3]: done

File: content/docs/0.18.x/define/quality-gates/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/0.18.x/define/quality-gates/index.md:9]: done

File: content/docs/0.18.x/define/quality-gates/index.md (hunk 3 above)
Comment (StackScribe) [reply content/docs/0.18.x/define/quality-gates/index.md:10]: Yup, we have a (nascent) reference page for SLO's.  Good catch -- I added a link to that page.

File: content/docs/0.18.x/define/quality-gates/index.md (hunk 5 above)
Comment (StackScribe) [reply content/docs/0.18.x/define/quality-gates/index.md:100]: done

**Review by thisthat — APPROVED** [review]

**Review by thschue — APPROVED** [review]

**Pull request merged**